pyproject.toml

```diff
@@ -45,6 +45,7 @@ dependencies = [
     "torch",
     "pyro-ppl",
     "graphviz",
+    "maq@git+https://github.com/grf-labs/maq.git@py0.2.0#egg=maq&subdirectory=python-package",
 ]
```

Comment:
I tried to make some documentation clarifications that are available if you change this to: `@py0.2.2`